onecodex/notebooks/exporters.py

```diff
@@ -190,6 +168,10 @@ class OneCodexPDFExporter(OneCodexHTMLExporter):
     export_from_notebook = "One Codex PDF Report"
     template_path = [ASSETS_PATH]
 
+    def __init__(self, config=None, **kw):
+        super(OneCodexPDFExporter, self).__init__(config=config, **kw)
```

Comment:
Since we're just supporting Python 3+ here, I think this could be simplified to just `super().__init__(config=config, **kw)`. They're basically the same, just a matter of brevity.

Reply:
I'm trying to keep this code a bit Python 2-y so we don't accidentally break stuff that _might_ work on Python 2, but fair point. I think I'll leave this as you _could_ use the report features technically in Python 2 I think (i.e., if you got all the deps installed and didn't need our Pandas integrations that are Python 3.4+ only).